### PR TITLE
Billing Screen

### PR DESCRIPTION
**Pull request by helloeddiepo**

### CODE REVIEW

File: packages/fan-fix-ui-controls/StyledTextInput.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ interface Props extends
     placeholder: string,
     type?: "text" | "number" | "password" | "textarea",
     value?: string,
-    variant?: Variant, 
+    variant: Variant,
```
Comment (maryseo070) [inline packages/fan-fix-ui-controls/StyledTextInput.tsx:23]: why make this required now? Are we not using primary as default?

File: packages/fan-fix/App.tsx [hunk 2]
```diff
@@ -68,6 +71,7 @@ export class App extends React.Component<Props> {
       <Router history={this.props.history}>
         <Switch>
           <Route path="/" exact component={this.props.authScreenProvider}/> {/* TODO set / path to home screen when ready, add routing/private paths */}
+          <Route path="/Billing" component={this.props.billingScreenProvider}/>
```
Comment (maryseo070) [inline packages/fan-fix/App.tsx:74]: nit: lower case path

File: packages/fan-fix/BillingScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    // Dummy Data
+    const hasCardOnFile = true;
+    const hasTransactions = false;
+
+    return (
+      <Screen navBarProps={navBarProps} tabBarProps={tabBarProps}>
+        <View styles={style.root}>
+          <View>
+            <View styles={style.paymentContainer}>
+              { hasCardOnFile ?
```
Comment (maryseo070) [inline packages/fan-fix/BillingScreen.tsx:104]: nit: line after {

File: packages/fan-fix/BillingScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+                  <>
+                    <StyledTextInput variant="default" type="text" placeholder="Name on card" value={this.nameOnCard} onChange={this.onNameOnCardChange} styles={style.textInput} />
+                    <StyledTextInput variant="default" type="text" placeholder="Stripe: Card input field placeholder" styles={style.textInput} /> {/* TODO: Placeholder for Stripe component */}
+                  </>
+                )
+              }
+            </View>
+
+            <View>
+              { hasTransactions ?
```
Comment (maryseo070) [inline packages/fan-fix/BillingScreen.tsx:126]: nit: line after {

File: packages/fan-fix/FinishProfileScreen.tsx [hunk 5]
```diff
@@ -154,35 +157,22 @@ export class FinishProfileScreen extends React.Component<Props> {
...
+              <StyledTextInput variant="default" type="textarea" placeholder="Bio" maxLength={bioMaxCharCount} value={this.bio} onChange={this.onBioChange} styles={style.textInput} />
+              <StyledTextInput variant="default" type="text" placeholder="Website link" value={this.websiteLink} onChange={this.onWebsiteLinkChange} styles={style.textInput} />
             </View>
 
             <View>
               { this.error && (
-                <View styles={style.errorMessageWrapper}>
-                  <StyledText variant="caption" styles={style.errorMessage}>{this.error}</StyledText>
-                </View>
+                <StyledText variant="caption" styles={style.errorMessage}>{this.error}</StyledText>
```
Comment (maryseo070) [inline packages/fan-fix/FinishProfileScreen.tsx:173]: lol eddie youre killin me with these conditional formats

Comment (helloeddiepo) [reply packages/fan-fix/FinishProfileScreen.tsx:173]: lol womp womp sorry

File: packages/fan-fix-ui-controls/StyledTextInput.tsx (hunk 1 above)
Comment (helloeddiepo) [reply packages/fan-fix-ui-controls/StyledTextInput.tsx:23]: Ah I was testing something and forgot to switch back. Good catch

**Review by mattryavec — APPROVED** [review]

**Pull request merged**